docs/design/2022-06-15-phyical plan explain.md

```diff
@@ -0,0 +1,406 @@
+# TiSpark Design Documents
+
+- Author(s): [Author Name](http://github.com/your-github-id), [Co-Author Name](http://github.com/your-github-id), ...
+- Tracking Issue: https://github.com/pingcap/tispark/issues/2385
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation or Background](#motivation-or-background)
+* [Detailed Design](#detailed-design)
+* [Test Design](#test-design)
+
+## Introduction
```

Comment:
Make the introduction easier to understand

Reply:
I change this description,PTAL